yellowbrick/text/tsne.py

```diff
@@ -130,43 +132,51 @@ class TSNEVisualizer(TextVisualizer):
     ax : matplotlib axes
         The axes to plot the figure on.
 
-    decompose : string or None
+    decompose : string or None, default: ``'svd'``
         A preliminary decomposition is often used prior to TSNE to make the
-        projection faster. Specify `"svd"` for sparse data or `"pca"` for
-        dense data. If decompose is None, the original data set will be used.
+        projection faster. Specify ``"svd"`` for sparse data or ``"pca"`` for
+        dense data. If None, the original data set will be used.
 
-    decompose_by : int
+    decompose_by : int, default: 50
         Specify the number of components for preliminary decomposition, by
         default this is 50; the more components, the slower TSNE will be.
 
-    classes : list of strings
+    labels : list of strings
```

Comment:
This is the biggest change; can now specify label names (unless they are detected from `y` on fit), classes are no longer filtered.